examples/test_sqg.py

```diff
@@ -5,9 +5,9 @@
 
 # the model object
 year = 1.
-m = sqg_model.SQGModel(L=2.*pi,nx=128, tmax = 100*year,
-        beta = 0., Nb = 1., H = 1., rek = 0., rd = None, dt = 0.01,
-                     taveint=year, use_fftw=True, ntd=8)
+m = sqg_model.SQGModel(L=2.*pi,nx=512, tmax = 40.005,
+        beta = 0., Nb = 1., H = 1., rek = 0., rd = None, dt = 0.005,
+                     taveint=1, ntd=8)
```

Comment:
I don't think 8 threads is appropriate for an example. We should probably not assume that users have a multicore machine.